src/app/GitUI/Hotkey/HotkeySettingsManager.cs

```diff
@@ -289,6 +289,8 @@ public IReadOnlyList<HotkeySettings> CreateDefaultSettings()
                     Hk(RevisionGridControl.Command.DeleteRef, Keys.Delete),
                     Hk(RevisionGridControl.Command.GoToChild, Keys.Control | Keys.N),
                     Hk(RevisionGridControl.Command.GoToCommit, Keys.Control | Keys.Shift | Keys.G),
+                    Hk(RevisionGridControl.Command.GoToFirstParent, Keys.Control | Keys.Left),
+                    Hk(RevisionGridControl.Command.GoToLastParent, Keys.Control | Keys.Right),
```

Comment:
Approving, excluding the assign of theses keys (same for RevDiff) - at least for now.

Reply:
`Ctrl+Left` and `Ctrl+Right` are text-editing hotkeys. That's why I have limited the hotkeys to the revision grid and to the file list.
I agree that there could come up a more important function for these hotkeys in these areas, too.
On the hand, they have never been used.
Removed.